<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(242, 236, 218)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,3,25,2">
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout3" stretch="2,1,2">
         <item>
          <widget class="QPushButton" name="btn_learn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>한컴 윤고딕 230</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	background-color: rgb(109, 73, 48);
}

QPushButton:hover{
          color: rgb(109, 73, 48);
          background: transparent;
          border: 2px solid brown;
          border-radius: 1px;
}</string>
           </property>
           <property name="text">
            <string>학습</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_mode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>한컴 윤고딕 230</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	background-color: rgb(109, 73, 48);
}

QPushButton:hover{
          color: rgb(109, 73, 48);
          background: transparent;
          border: 2px solid brown;
          border-radius: 1px;
}</string>
           </property>
           <property name="text">
            <string>모드 선택</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_quiz">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>109</red>
                 <green>73</green>
                 <blue>48</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>한컴 윤고딕 230</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: white;
	background-color: rgb(109, 73, 48);
}

QPushButton:hover{
          color: rgb(109, 73, 48);
          background: transparent;
          border: 2px solid brown;
          border-radius: 1px;
}</string>
           </property>
           <property name="text">
            <string>퀴즈</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout1">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>한컴 윤고딕 240</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(164, 176, 116)</string>
           </property>
           <property name="text">
            <string>딥러닝 기반 수어 교육</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout2" stretch="1,1">
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout2_1" stretch="1,3,1,1">
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="label_text_1">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>문제 1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_word1">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>단어</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_check1">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>O</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_video1">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: white;
	background-color: rgb(66, 65, 6);
}

QPushButton:hover{
          color: rgb(66, 65, 6);
          background: transparent;
          border: 2px solid dark green;
          border-radius: 1px;
}</string>
               </property>
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout2_2" stretch="1,3,1,1">
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="label_text">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>문제 2</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_word2">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>단어</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_check2">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>O</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_video2">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: white;
	background-color: rgb(66, 65, 6);
}

QPushButton:hover{
          color: rgb(66, 65, 6);
          background: transparent;
          border: 2px solid dark green;
          border-radius: 1px;
}</string>
               </property>
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout2_3" stretch="1,3,1,1">
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="label_text">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>문제 3</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_word3">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>단어</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_check3">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>O</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_video3">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: white;
	background-color: rgb(66, 65, 6);
}

QPushButton:hover{
          color: rgb(66, 65, 6);
          background: transparent;
          border: 2px solid dark green;
          border-radius: 1px;
}</string>
               </property>
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout2_4" stretch="1,3,1,1">
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>50</number>
             </property>
             <item>
              <widget class="QLabel" name="label_text">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>문제 4</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_word4">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>단어</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_check4">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>O</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_video4">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: white;
	background-color: rgb(66, 65, 6);
}

QPushButton:hover{
          color: rgb(66, 65, 6);
          background: transparent;
          border: 2px solid dark green;
          border-radius: 1px;
}</string>
               </property>
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout2_5" stretch="1,3,1,1">
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>50</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_text">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>문제 5</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_word5">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>단어</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_check5">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>O</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_video5">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: white;
	background-color: rgb(66, 65, 6);
}

QPushButton:hover{
          color: rgb(66, 65, 6);
          background: transparent;
          border: 2px solid dark green;
          border-radius: 1px;
}</string>
               </property>
               <property name="text">
                <string>-&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>점수 : </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_score">
               <property name="font">
                <font>
                 <family>한컴 윤고딕 230</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>5/5</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="lb_video">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
